sdk/schemaregistry/azure-schemaregistry/swagger/README.md

```diff
@@ -19,5 +19,5 @@ clear-output-folder: true
 python: true
 add-credential: true
 credential-scopes: "https://eventhubs.azure.net/.default"
-package-version: "1.0.0b3"
+package-version: "1.0.0b4"
```

Comment:
```suggestion
package-version: "1.0.0"
```

Reply:
changes here: https://github.com/Azure/azure-sdk-for-python/pull/21573